build/test-aot-compatibility.ps1

```diff
@@ -1,13 +1,16 @@
 param([string]$targetNetFramework)
 
+$runtime = $IsWindows ? "win-x64" : "linux-x64"
+$app = $IsWindows ? "./OpenTelemetry.AotCompatibility.TestApp.exe" : "./OpenTelemetry.AotCompatibility.TestApp"
+
 $rootDirectory = Split-Path $PSScriptRoot -Parent
-$publishOutput = dotnet publish $rootDirectory/test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj -nodeReuse:false /p:UseSharedCompilation=false
+$publishOutput = dotnet publish $rootDirectory/test/OpenTelemetry.AotCompatibility.TestApp/OpenTelemetry.AotCompatibility.TestApp.csproj --runtime $runtime -nodeReuse:false /p:UseSharedCompilation=false
```

Comment:
I was passing it explicitly so it married-up obviously with the output path, as the binary is found later down to run what was compiled, and I thought that made it easier to follow for the casual observer.

Reply:
>  it married-up obviously with the output path

Following that same logic you would be passing `-f $targetNetFramework` to publish, because we are using it in the output path. (not that I'm suggesting we actually do that as well).

I'm just of the opinion to not clutter up the command if it doesn't add value. It just adds more things for people to try to understand what this line is doing.